src/deluge/model/clip/instrument_clip.cpp

```diff
@@ -1306,6 +1308,7 @@ void InstrumentClip::nudgeNotesVertically(int32_t change, ModelStackWithTimeline
 			NoteRow* thisNoteRow = noteRows.getElement(i);
 			// transpose by semitones or by octave
 			thisNoteRow->y += change;
+			semitonesNudged += change;
```

Comment:
This isn't actually very useful info. Display the number of scale degrees it's been nudged instead